scenarios/detection/02_mask_rcnn.ipynb

```diff
@@ -213,7 +213,7 @@
    "source": [
     "Masks are grayscale images where the value of pixels belong to a specific object are the object id indexed from 1.\n",
     "\n",
-    "**NOTE**: When preparing the dataset, we must make sure the order of objects in the mask image be the same as in the annotation file.  In other words, for the following image, the value of `water_bottle`'s pixels must be 1 and that of `milk_bottle` must be 2."
+    "**NOTE**: When preparing the dataset, we must make sure the order of objects in the mask image be the same as in the annotation file.  In other words, for the following image, the value of `water_bottle`'s pixels must be 1 and that of `milk_bottle` must be 2.  In preparing the dataset, we used [Labelbox](https://labelbox.com/) for mask annotation.  The masks from Labelbox's annotation can be extracted by using our `extract_masks_from_labelbox_json()`."
```

Comment:
Don't think it's clear right now where "extract_masks_from_labelbox_json()" is. Could add: "The masks from Labelbox's annotation can be extracted by using our `extract_masks_from_labelbox_json()` WHICH CAN BE IMPORTED FROM UTILS_CV.DETECTION.DATA"

Reply:
Done